lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1083,6 +1086,12 @@ def sanitize_network_params(self):
                             str(dvp.config.defaultPortConfig.vlan.vlanId) == network['vlan']:
                         network['name'] = dvp.config.name
                         break
+                    if 'dvswitch_name' in network and \
+                            dvp.config.distributedVirtualSwitch.name == network['dvswitch_name'] and \
+                            dvp.config.name == network['vlan']:
```

Comment:
should this be `network['name']`? I thought `network['vlan']` was the vlan id `int`?